cmd/kaspawallet/balance.go

```diff
@@ -22,10 +30,13 @@ func balance(conf *balanceConfig) error {
 		return err
 	}
 
-	fmt.Printf("Balance:\t\tKAS %f\n", float64(response.Available)/constants.SompiPerKaspa)
-	if response.Pending > 0 {
-		fmt.Printf("Pending balance:\tKAS %f\n", float64(response.Pending)/constants.SompiPerKaspa)
+	println("Address                                                                       Available             Pending")
```

Comment:
I would make the detailed format optional, and the default view should be only balance and pending balance IMHO